subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/MissingFileSnapshot.java

```diff
@@ -29,11 +29,19 @@
     private static final HashCode SIGNATURE = Hashing.signature(MissingFileSnapshot.class);
 
     public MissingFileSnapshot(String absolutePath, String name) {
-        super(absolutePath, name);
+        super(absolutePath, name, false);
```

Comment:
This should call through to `this` not `super`.

Reply:
Done in https://github.com/gradle/gradle/issues/13053.